### DISCUSSION TITLE
Can you run Langflow, (embeddings and LLM's) on CUDA?

### DISCUSSION THREAD

**Discussion by rodgermoore** [langflow-ai/langflow#318] [Q&A]
Can you run Langflow, (embeddings and LLM's) on CUDA? If so, where can I find instructions to run this on Docker?

Thanks!

**Comment by carlosrcoelho** [comment]
The process depends on the embeddings. If the user is utilizing Hugging Face embeddings for inference, they can run the inference engine on a CUDA-based server. However, it is completely independent of Langflow. We simply utilize the base URLs or API URLs of the deployed services. To my knowledge, Langflow does not have any resources that directly use CUDA. We have transitioned most services to use API examples, like Ollama and Hugging Face inference.
